testsuite/tests/ui/conftest.py

```diff
@@ -379,6 +384,52 @@ def get_resultsdir_path(node):
     return path
 
 
+def fullpage_screenshot(driver, file_path):
+    """
+        A full-page screenshot function. It scroll the website and screenshots it.
+        - Creates multiple files
+        - Screenshots are made only vertically (on Y axis)
+    """
+    if driver is None:
+        return
+
+    # Removal of the height: 100% style, that disables scroll.
+    driver.execute_script("document.body.style.height = 'unset'")
+    driver.execute_script("document.body.parentNode.style.height = 'unset'")
+
+    total_height = driver.execute_script("return document.body.parentNode.scrollHeight")
+    viewport_height = driver.execute_script("return window.innerHeight")
+
+    dummy_file = file_path + "/dummy_img.png"
+    driver.get_screenshot_as_file(dummy_file)
+    screenshot = Image.open(dummy_file)
+    width, height = screenshot.size
+    del screenshot
+    os.remove(dummy_file)
+
+    scaling_constant = (float(height) / float(viewport_height))
+    stitched_image = Image.new('RGB', (width, int(total_height * scaling_constant)))
+    part = 0
+
+    for scroll in range(0, total_height, viewport_height):
+        driver.execute_script("window.scrollTo({0}, {1})".format(0, scroll))
+        file_name = file_path + "part_{0}.png".format(part)
+        driver.get_screenshot_as_file(file_name)
```

Comment:
How about using `driver.get_screenshot_as_png()` and get rid of the temporary files?

Reply:
An awesome idea (forgot that this function/method exists). Changed in a latest commit!